test/GCClassic/README.md

```diff
@@ -0,0 +1,97 @@
+
+Parallelization tests are similar to integration tests, except that each simulation is run twice with a different number of cores. The restart files from each run then are compared for bitwise-identicality.
+
+## Files
+
+### Integration Test Scripts
+
+- `intTest.sh`
+  - Driver script to run GEOS-Chem Classic integration tests
+- `intTestCreate.sh`
+  - Script to create GEOS-Chem Classic run directories for integration tests
+- `intTestCompile.sh`
+  - Script to compile GEOS-Chem Classic executables
+- `intTestExecute.sh`
+   - Script to run GEOS-Chem Classic integration test simulations
+
+### Parallelization Test Scripts
+
+- `parTest.sh`
+  - Driver script to run GEOS-Chem Classic parallelization tests
+- `parTestCreate.sh`
+  - Script to create GEOS-Chem Classic run directories for parallelization tests
+- `parTestCompile.sh`
+  - Script to compile GEOS-Chem Classic executables
+- `parTestExecute.sh`
+   - Script to run GEOS-Chem Classic parallelization test simulations
+
+### Shared Scripts
+
+- `commonFunctionsForTests.sh`
+  - Link to `../shared/commonFunctionsForTests.sh`, which contains global variables and functions for the integration and parallelization tests.
+
+## Running GEOS-Chem Classic Integration and/or Parallelization Tests
+
+Before you submit any GEOS-Chem Classic integration or parallelization tests, take a moment and make sure that:
+
+1. The `GCClassic` superproject is checked out to the correct branch and commit.
+2. The `GEOS-Chem` submodule is checked out to the correct branch and commit.
+3. The `HEMCO` submodule is checked out to the correct branch and commit.
+
+### With the SLURM scheduler
+
+```console
+$ cd test/GCClassic
+$ ./intTest.sh -d /path/to/test/dir -e /path/to/env-file -s -p partition   # Integration tests
+$ ./parTest.sh -d /path/to/test/dir -e /path/to/env-file -s -p partition   # Parallelization tests
+```
+
+### With the LSF scheduler
+
+```console
+$ cd test/GCClassic
+$ ./intTest.sh -d /path/to/test/dir -e /path/to/env-file -l -p partition   # Integration tests
+$ ./parTest.sh -d /path/to/test/dir -e /path/to/env-file -l -p partition   # Parallelization tests
+```
+
+### Interactively at the command line
+
+```console
+$ cd test/GCClassic
+$ ./intTest.sh -d /path/to/test/dir -e /path/to/env-file   # Integration tests
+$ ./parTest.sh -d /path/to/test/dir -e /path/to/env-file   # Parallelization tests
+```
+
+## Command-line Arguments
+
+The integration test scripts and parallelization test scripts accept the following command-line arguments
+
+`-d /path/to/test/dir` specifies the root directory where integration/parallelization test subdirectories and scripts will be placed.
+
+`-e /path/to/env-file` specifies the file that is used to initialize the software environment.  This file will typically contain either `module load` or `spack load` commands.
+
+`-p partition` specifies the partition for the SLURM or LSF scheduler.  You may omit this for running interactively
+
```

Comment:
Could you add descriptions of the `-s` and `-l` options here too?

Reply:
This is now done in commit 79b6cb5643279e709052a00eb9a82f18d6f47019